build/e2e_test.go

```diff
@@ -815,6 +815,16 @@ func TestScoring(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	examplePython, err := os.ReadFile("./testdata/example.py")
```

Comment:
In a follow-up, I'll try to pull in SCIP ctags so we can run the same tests using that binary.